README.autotools

```diff
@@ -0,0 +1,167 @@
+-*- coding: utf-8-unix -*-
+
+				 Libsv
+				 =====
+
+
+Topics
+------
+
+  1. Introduction
+  2. License
+  3. Install
+  4. Usage
+  A. Credits
+  B. Bugs, vulnerabilities and contributions
+  C. Resources
+  D. Badges and static analysis
+
+
+1. Introduction
+---------------
+
+This  package  installs  a  C  language  library  implementing  semantic
+versioning for the C language.
+
+  This README file  describes the building infrastructure  using the GNU
+Autotools.
+
+
+2. License
+----------
+
+See the LICENSE file.
+
+
+3. Install
+----------
+
+To install from a proper release tarball, do this:
+
+    $ cd libsv-3.1
```

Comment:
I got that version number from a CMake file; I do not know CMake, so it was  a mistake.  What is the version number then?